frame/election-provider-multi-phase/src/signed.rs

```diff
@@ -52,6 +52,8 @@ pub struct SignedSubmission<AccountId, Balance: HasCompact, Solution> {
 	pub raw_solution: RawSolution<Solution>,
 	/// The reward that should potentially be paid for this solution, if accepted.
 	pub reward: Balance,
+	// The estimated fee `who` paid to submit the solution.
+	pub call_fee: Balance,
```

Comment:
`reward` was a balance, and `call_fee` is a balance, so this should decode the same through runtime upgrades.